CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 - UniqueId class for unique IDs within a process [#1010](https://github.com/eclipse-iceoryx/iceoryx/issues/1010)
 - Add requirePublisherHistorySupport option at subscriber side (if set to true requires historyRequest <= historyCapacity to be eligible for connection) [#1029](https://github.com/eclipse-iceoryx/iceoryx/issues/1029)
 - Add `/tools/scripts/ice_env.sh` shell script to provide simple access to docker containers for CI debugging [#1049](https://github.com/eclipse-iceoryx/iceoryx/issues/1049)
+- Add C++17 `std::perms` as `cxx::perms` to iceoryx hoofs. [#1059](https://github.com/eclipse-iceoryx/iceoryx/issues/1059)
```

Comment:
A short example or explanation link would be good in the API breaking changes section.

Reply:
@dkroenke I added the path to the file in the changelog so that it can be found easily.
But in my opinion we do not have any kind of API breaking in this PR since we only add the `perms` enum.

This API breaking comes with the shared memory PR #1053 . But here we break private API since the `shared_memory.hpp` is located inside of `internal`.